<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProfileDialog</class>
 <widget class="QDialog" name="ProfileDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>557</width>
    <height>386</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLineEdit" name="lineProfileFilter">
       <property name="placeholderText">
        <string>Search for profile ...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cmbProfileTypes"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="ProfileTreeView" name="profileTreeView">
     <property name="rootIsDecorated">
      <bool>false</bool>
     </property>
     <property name="uniformRowHeights">
      <bool>true</bool>
     </property>
     <property name="itemsExpandable">
      <bool>false</bool>
     </property>
     <property name="sortingEnabled">
      <bool>false</bool>
     </property>
     <property name="headerHidden">
      <bool>false</bool>
     </property>
     <property name="expandsOnDoubleClick">
      <bool>false</bool>
     </property>
     <attribute name="headerVisible">
      <bool>true</bool>
     </attribute>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0,0,0,0">
     <item>
      <widget class="StockIconToolButton" name="newToolButton">
       <property name="toolTip">
        <string>Create a new profile using default settings.</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="StockIconToolButton" name="deleteToolButton">
       <property name="toolTip">
        <string>Remove this profile. System provided profiles cannot be removed.</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="StockIconToolButton" name="copyToolButton">
       <property name="toolTip">
        <string>Copy this profile.</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnImport">
       <property name="text">
        <string>Import</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Help|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>StockIconToolButton</class>
   <extends>QToolButton</extends>
   <header>widgets/stock_icon_tool_button.h</header>
  </customwidget>
  <customwidget>
   <class>ProfileTreeView</class>
   <extends>QTreeView</extends>
   <header>widgets/profile_tree_view.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ProfileDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ProfileDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
